docs/systemd.md

```diff
@@ -25,10 +25,10 @@ Each service has its configuration file located in `/etc/default`. Default
 configurations can be found [here](../docker/package/defaults/).
 
 Files created by the services will be located in `/var/lib/tezos/` by default.
-`tezos-{accuser, baker, endorser}-<protocol>` services can have configurable
+`tezos-{accuser, baker}-<protocol>` services can have configurable
 data directory.
 
-`tezos-{accuser, endorser}` have configurable node address, so that they can be used with both
+`tezos-accuser` has a configurable node address, so that they can be used with both
```

Comment:
```suggestion
`tezos-accuser` has a configurable node address, so that it can be used with both
```

? Otherwise I don't understand what's `they` in this